src/main/resources/templates/moves-by-type.html

```diff
@@ -11,13 +11,82 @@
 
 <div class="govuk-width-container" layout:fragment="content" th:inline="text">
 
-    <div th:replace="fragments/summary.html :: summary"></div>
+    <div class="govuk-breadcrumbs ">
+        <ol class="govuk-breadcrumbs__list">
+            <li class="govuk-breadcrumbs__list-item">
+                <a class="govuk-breadcrumbs__link" href="/dashboard" th:inline="text">[[${#dates.monthName(startOfMonthDate)}]]
+                    [[${#dates.format(startOfMonthDate, 'yyyy')}]]</a>
+            </li>
+            <li class="govuk-breadcrumbs__list-item">
+                <span class="govuk-breadcrumbs__link" th:inline="text">[[${moveType}]] moves</span>
+            </li>
+        </ol>
+    </div>
 
-    <section class="mt5">
-        <h2 class="govuk-heading-m pt1 bt bw2">Moves by type</h2>
-        <div th:each="move: ${moves}">
-            <div th:replace="fragments/list-moves-by-type.html :: moves"></div>
+    <div class="flex justify-between items-end mv4">
+        <div class="flex-auto">
+            <h1 class="govuk-heading-xl ma0" th:inline="text">[[${moveType}]]</h1>
         </div>
+        <div>
+            <a th:href="@{|/generate-prices-spreadsheet/|}"
+               th:attr="download=${'moves-' + '.csv'}"
+               class="ma0 govuk-link govuk-heading-s download-icon">Download all moves</a>
+        </div>
+    </div>
+
+    <div th:replace="fragments/move-summary.html :: summary"></div>
+
+    <section class="mt3">
+
+        <table class="govuk-table">
+            <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th class="govuk-table__header" scope="col" aria-sort="none" width="80">Move ID</th>
+                <th class="govuk-table__header" scope="col" aria-sort="none">Pickup location</th>
+                <th class="govuk-table__header" scope="col" aria-sort="none">Pickup type</th>
```

Comment:
Minor comment screenshot in ticket shows this heading as **Location type** and not **Pickup type**.